src/components/SingleArticle.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticle} from "../api";
+
+
+const SingleArticle = () => {
+  const [article, setArticle] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
You've set up the isLoading pattern, but you're not actually using it at the moment! Please add some conditional logic above your return statement to give the user some feedback when the data is being loaded.

Reply:
Please also add an isError pattern - to show the user an error message if they try to request an article id that doesn't exist!